searchsummary/src/vespa/searchsummary/docsummary/linguistics_tokens_converter.h

```diff
@@ -0,0 +1,37 @@
+// Copyright Vespa.ai. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+
+#pragma once
+
+#include "i_string_field_converter.h"
+
+namespace document {
+
+class FieldValue;
```

Comment:
Are these needed? If not, please remove.

Reply:
Removed.